### ISSUE TITLE
JSCS fails... a lot

### ISSUE THREAD

**Issue by sdaitzman** [yrsegal/LaserQueue#104]
Adding an issue for tracking this as a mini style-refactor

**Comment by sdaitzman** [comment]
_config.js issues fixed in https://github.com/yrsegal/LaserQueue/commit/26e5b16001eec60de71df546491954ff94270962 partially but not entirely because Multiline Strings

**Comment by sdaitzman** [comment]
_socketSetup.js issues fixed in 3224c1e

**Comment by sdaitzman** [comment]
_init.js issues fixed in c645efb

**Comment by sdaitzman** [comment]
_functions.js issues fixed in f41decd (god rest my poor soul)

**Comment by yrsegal** [comment]
`god rest my poor soul`  
nah, you'll be refactoring code eternally

**Comment by sdaitzman** [comment]
so, options for multiline strings in this refactor:
- leave them the way they are and manually ignore them (no)
- add an ES6 transpiler to use backticks ( ` ) as the quotation marks (giant pain and not pretty)
- concatenate the strings by putting single-quotes around each line and a + at the end (goddammit, but this is the best solution I think)

**Comment by yrsegal** [comment]
all teh nuuuuuu

most of them can use \n tho

**Comment by yrsegal** [comment]
if you're only using the multiline to add newlines, you can use \n. Otherwise, + concats.

**Comment by sdaitzman** [comment]
It's largely for hacky things like

```
doStuffWith('
  <tag>
    <tagInside> askldmakmds </tagInside>
  </tag>
');

```

Which will now be:

```
doStuffWith(
  '<tag>' +
    '<tagInside> askldmakmds </tagInside>' +
  '</tag>'
);

```

**Comment by yrsegal** [comment]
\n solves all le thangs

**Comment by sdaitzman** [comment]
Closed by c95cc5a and 30a7aa9 !
